agent/util-scripts/pbench-tool-meister-start

```diff
@@ -603,6 +632,42 @@ port {redis_port:d}
                 )
 
 
+def terminate_no_wait(
+    group: ToolGroup, logger: logging.Logger, redis_client: redis.Redis, key: str
+) -> None:
+    """
+    Use a low-level Redis publish operation to send a "terminate" request to
+    all clients without waiting for a response. This is intended to be used
+    as part of cleanup during unexpected termination, to make at least an
+    attempt at clean termination before quitting. Success is not guaranteed,
+    and we only check whether the message was sent.
+
+    TODO: Ideally, we'd wait some reasonable time for a response from TDS and
```

Comment:
The publish API should return 0 if nobody was listening, or a non-zero count of entities that were subscribed to the channel that received the message.

If that is true, do we still need this comment?

Reply:
My concern was more that a client might receive the message but fail to respond in a reasonable period of time.